docs/backups.md

```diff
@@ -0,0 +1,23 @@
+# OSBS Backups
+
+## Creating Backups
+
+Creating OSBS backups is very simple. To backup all relevant data, one needs to:
+
+1. Pause OSBS by `osbs pause-builds`
+2. Wait for all running builds to finish
```

Comment:
`osbs pause-builds` already performs step 2. Do we just need to say "..., which will prevent new builds being scheduled and wait for running builds to finish" in step 1?